src/Test/Mock/DeferredApiHelper.test.ts

```diff
@@ -101,3 +101,82 @@ test("GIVEN DeferredApiHelper WHEN resolveRequest THEN resolves first matching r
     ),
   ).toThrow();
 });
+
+test("GIVEN BaseApiHelper POST WHEN response contains large integers THEN integers are converted to bigints", async () => {
+  const apiHelper = new DeferredApiHelper();
+
+  //POST
+  apiHelper.post<{ foo: bigint }, { foo: bigint }>("", "", {
+    foo: 9223372036854775807n,
+  });
+  apiHelper.pendingRequests;
+  expect(apiHelper.pendingRequests[0]).toEqual({
+    body: {
+      foo: 9223372036854775807n,
```

Comment:
maybe a stupid question, but can we send the request with the "n" at the end of the BigInt ? Is that supported ?